core/src/main/java/com/github/mizool/core/rest/errorhandling/ErrorHandler.java

```diff
@@ -151,6 +169,12 @@ private void recordViolation(ConstraintViolation<?> violation, ListMultimap<Stri
         target.put(propertyName, errorDto);
     }
 
+    private void recordRuleViolation(RuleViolation violation, ListMultimap<String, ErrorDto> target)
```

Comment:
move this up so it follows `handleRuleViolationError()`.

Reply:
for consistency, also do the following:

- adapt the position of `recordViolation()` accordingly
- rename `recordViolation()` to `recordConstraintViolation()`
- rename `handleValidationError()` to `handleConstraintViolationError()`